quality_control_oca_timesheet/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Patryk Pyczko (APSL-Nagarro)<ppyczko@apsl.net>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Quality Control - Timesheet (OCA)",
+    "version": "17.0.1.0.0",
+    "category": "Quality Control",
+    "website": "https://github.com/OCA/manufacture",
+    "author": "Patryk Pyczko, Odoo Community Association (OCA)",
```

Comment:
Here goes the company, not the contributor

Reply:
Thank you for letting me know, changed!